src/konfetti/vault/core.py

```diff
@@ -141,9 +148,8 @@ def disabled(self):
         # Move to some global config or we need to modify in runtime?
         return EnvVariable(SECRETS_DISABLED_VARIABLE, default=False, cast=bool).evaluate()
 
-    @property
-    def override_variable_name(self):
-        # type: () -> str
+    def override_variable_name(self, add_keys=None):
```

Comment:
Should it be `False`? otherwise, the type should be `Optional[bool]`, but I assume it is a binary flag and `None` is not needed

Reply:
You're right, makes more sense.